### ISSUE TITLE
Disabling wood type variants should disable the oak wood recipe

### ISSUE THREAD

**Issue by FluffiKiki** [svenhjol/Charm#189]
When I disable the recipes to disable the vanilla wood variants of the barrel and bookshelf, it removes the items themselves, but the recipe defaults to the oak recipe only. I remove these features so that I can use woods of other types to make these items in case vanilla logs are nowhere to be found, but I can't do that while Charm is on. Could you consider a config for this feature? Thanks.

**Comment by svenhjol** [comment]
I don't follow. Could you write a list of steps for me to repeat?

**Comment by FluffiKiki** [comment]
So I actually found that it doesn't disable the recipes altogether. I will be using pictures and the barrels option to demonstrate. 
1) I set the config option "AllTheBarrels" to false 
![2019-11-16_LI](https://user-images.githubusercontent.com/42556991/68999605-543bc880-0888-11ea-98d6-c59ee9c0e8c9.jpg)

2) For the purpose of this demo, I started a new world. 
3) In Picture 2, the barrels still appear and, according to JEI, are still craftable. (The second barrel is from another mod)
![2019-11-16 (1)](https://user-images.githubusercontent.com/42556991/68999608-6289e480-0888-11ea-833d-a97454ae7ab5.png)

This is on 1.14.4, version 1.4.0. (From the version I reported on). I wanted to report it because the vanilla barrels are craftable from all types of wood, to eliminate the frustration of having to stick to one type of wood per barrel.
